.github/workflows/windows.yml

```diff
@@ -66,6 +66,12 @@ jobs:
         run:  ./scripts/log-env.sh
       - name:  Debug Build
         shell: python scripts\msys-bash.py {0}
-        run:   ./scripts/build.sh --compiler ${{ matrix.compiler }} --build-type Debug --bin-path /mingw${{ matrix.bits }}/bin
+        env:
+          ARCH_32: i686
+          ARCH_64: x86_64
+        run:  |
+          export MSYSTEM_CARCH="$ARCH_${{ matrix.bits }}"
+          export MSYSTEM_CHOST="$ARCH_${{ matrix.bits }}-pc-msys"
```

Comment:
Perhaps we could simplify it after we'll turn the matrix into an enumeration of build parameters, but as for now - this looks fine :)

Reply:
> we could simplify it after we'll turn the matrix into an enumeration of build parameters
Absolutely!